OpenDreamClient/Interface/Controls/ControlBrowser.cs

```diff
@@ -89,6 +89,7 @@ public ControlBrowser(ControlDescriptor controlDescriptor, ControlWindow window)
 
                     // Reassemble the query params without element then convert to winset syntax
                     var query = queryParams.ToString();
+                    query = HttpUtility.UrlDecode(query);
```

Comment:
Shouldn't this be placed below the `Replace('&', ';')` call? Could break winsets with `&` in them.

Reply:
I don't think so? Placing it below that will turn e.g. `&gt;` into `;gt;` first so it doesn't get URL decoded

Reply:
Ah, true. File that under the "more robust parsing" then.